order/buyer/download-document.md

```diff
@@ -0,0 +1,5 @@
+---
+description: How to download a document from an order or line response
+---
+
+# Download order response document
```

Comment:
Is this still a WIP?

Reply:
Yes, I will add this under TC-5385 Create API v2 manual for buyer order process (part 2)